charts/gateway-helm/templates/namespace.yaml

```diff
@@ -0,0 +1,6 @@
+{{ if .Values.createNamespace }}
```

Comment:
maybe I'm out of date, I recall it's not a good practice to put namespace in a helm chart.

Reply:
it is not, but we are using `helm` as the source of truth for manifest templates, in the previous release it was `customize`
* for the common case using `helm install` we skip this file (`createNamespace` is `false`)
* for the manifest generation case to support install via 1 single YAML, we rely on it